archetypes/msf4j-microservice-archetype/src/main/resources/archetype-resources/src/main/java/MicroService.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ${package};
+
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+
+/**
+ * This is the MicroService resource class.
+ * See <a href="https://github.com/wso2/msf4j#getting-started">https://github.com/wso2/msf4j#getting-started</a>
+ * for the usage of annotations.
+ *
+ * @since ${version}
+ */
+@Path("/microservice")
+public class MicroService {
+
+    @GET
+    @Path("/get")
+    public String get() {
+        // TODO: Implementation for HTTP GET request
+        return "Hello";
+    }
+
+    @POST
+    @Path("/post")
+    public void post() {
```

Comment:
I thought of that but in that case we will need to add a bean class in the archetype which could make this less generic. I think we should keep it this way and allow the user modify it once he creates an actual project from the archetype.